backpack/extensions/firstorder/batch_grad/batch_grad_base.py

```diff
@@ -1,17 +1,40 @@
+"""Passes the calls for the parameters to the derivatives class."""
 from backpack.extensions.firstorder.base import FirstOrderModuleExtension
 
 
 class BatchGradBase(FirstOrderModuleExtension):
+    """Passes the calls for the parameters to the derivatives class.
+
+    This class implements functions with method names from params.
+
+    If child class wants to overwrite these methods
+    - for example to support an additional external module -
+    it can do so using the interface for parameter "param1"
+    param1(ext, module, g_inp, g_out, bpQuantities):
+        return batch_grads
+    In this case, the method is not overwritten by this class.
+    """
+
     def __init__(self, derivatives, params=None):
+        """Initializes all methods.
+
+        If the param method has already been defined, it is left unchanged.
+
+        Args:
+            derivatives(backpack.core.derivatives.basederivatives.BaseParameterDerivatives):# noqa
+                Derivatives object assigned to self.derivatives.
+            params (list[str]): list of strings with parameter names. Defaults to None.
+        """
```

Comment:
The documentation is quite general. I think it would be easier to understand if it was referring more specifically to the `BatchGrad` extension.

Reply:
I changed the documentation such that it says which quantities are calculated. If you mean something different by "specifically to the BatchGrad extension", please explain it to me.

Reply:
Maybe something along [these lines](https://gist.github.com/redlotus/3bc387c2591e3e908c9b63b97b11d24e#file-docstrings-py-L270-L274) works?